internal/packager/cnab.go

```diff
@@ -81,7 +81,8 @@ func ToCNAB(app *types.App, invocationImageName string) (*bundle.Bundle, error)
 		Parameters:  parameters,
 		Actions: map[string]bundle.Action{
 			internal.Namespace + "inspect": {
-				Modifies: false,
+				Modifies:  false,
+				Stateless: true,
```

Comment:
I suppose/hope this means you can now drop the dummy creds from `runInspect`?

Reply:
Right! I forgot this one, thanks for spotting it!

Reply:
Fixed